proto/executor.proto

```diff
@@ -127,11 +127,14 @@ message ANNQueryInfo {
 	optional VectorDistanceMetric distance_metric = 2 [(gogoproto.nullable) = false];
 	optional uint32 top_k = 3 [(gogoproto.nullable) = false];
 	optional string column_name = 4 [(gogoproto.nullable) = false]; // For debug purpose only. Currently only used in explain.
-	optional int64 column_id = 5 [(gogoproto.nullable) = false];
+	optional int64 column_id = 5 [(gogoproto.nullable) = true];
```

Comment:
Is it ok to turn this into a nullable field? @breezewish @zimulala

Reply:
I think TiFlash and TiDB using the correct tipb version should work just fine. I am not sure.  @breezewish please help to confirm it

Reply:
@breezewish PTAL